src/Compilers/CSharp/Test/Semantic/IOperation/ioperationtests_isymbolinitializer.cs

```diff
@@ -0,0 +1,270 @@
+            Assert.Equal(2, nodes.Length);
+
+            var semanticModel = compilation.GetSemanticModel(tree);
+            foreach (var node in nodes)
+            {
+                Assert.Null(semanticModel.GetOperationInternal(node));
+            }
+        }
+
+        [Fact, WorkItem(17595, "https://github.com/dotnet/roslyn/issues/17595")]
+        public void ConstantInitializers()
+        {
+            var source = @"
+class C
+{
+    static int s1 = 1;
+    int i1 = 1;
+    void M(int p1 = 0, params int[] p2 = null) { }
+}";
+
+            var compilation = CreateCompilationWithMscorlib(source, options: TestOptions.ReleaseDll, parseOptions: TestOptions.Regular);
+
+            var tree = compilation.SyntaxTrees.Single();
+            var nodes = tree.GetRoot().DescendantNodes().OfType<EqualsValueClauseSyntax>().ToArray();
+            Assert.Equal(4, nodes.Length);
+
+            compilation.VerifyOperationTree(nodes[0], expectedOperationTree:
+@"IFieldInitializer (Field: System.Int32 C.s1) (OperationKind.FieldInitializerAtDeclaration)
+  ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1)");
+
+            compilation.VerifyOperationTree(nodes[1], expectedOperationTree:
+@"IFieldInitializer (Field: System.Int32 C.i1) (OperationKind.FieldInitializerAtDeclaration)
+  ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1)");
+
+            compilation.VerifyOperationTree(nodes[2], expectedOperationTree:
+@"IParameterInitializer (Parameter: [System.Int32 p1 = 0]) (OperationKind.ParameterInitializerAtDeclaration)
+  ILiteralExpression (Text: 0) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 0)");
+
+            compilation.VerifyOperationTree(nodes[3], expectedOperationTree:
+@"IParameterInitializer (Parameter: params System.Int32[] p2) (OperationKind.ParameterInitializerAtDeclaration)
+  IConversionExpression (ConversionKind.Cast, Implicit) (OperationKind.ConversionExpression, Type: System.Int32[], Constant: null)
+    ILiteralExpression (Text: null) (OperationKind.LiteralExpression, Type: null, Constant: null)");
+        }
+
+
+        [Fact, WorkItem(17595, "https://github.com/dotnet/roslyn/issues/17595")]
+        public void ExpressionInitializers()
+        {
+            var source = @"
+class C
+{
+    static int s1 = 1 + Foo();
+    int i1 = 1 + Foo();
+
+    static int Foo() { return 1; }
+}";
+
+            var compilation = CreateCompilationWithMscorlib(source, options: TestOptions.ReleaseDll, parseOptions: TestOptions.Regular);
+
+            var tree = compilation.SyntaxTrees.Single();
+            var nodes = tree.GetRoot().DescendantNodes().OfType<EqualsValueClauseSyntax>().ToArray();
+            Assert.Equal(2, nodes.Length);
+
+            compilation.VerifyOperationTree(nodes[0], expectedOperationTree:
+@"IFieldInitializer (Field: System.Int32 C.s1) (OperationKind.FieldInitializerAtDeclaration)
```

Comment:
here is link - http://index/?query=CodeActionsUnitTestGenerator.cs&rightProject=RoslynETAHost&file=Diagnostics%5CFeatures%5CCodeActions%5CCodeActionsUnitTestGenerator.cs&rightSymbol=ezuztzshsthj